src/sbt-test/sbt-reactive-app/lagom-endpoints/build.sbt

```diff
@@ -0,0 +1,56 @@
+name := "lagom-endpoints"
+scalaVersion in ThisBuild := "2.11.12"
+
+
+enablePlugins(SbtReactiveAppPlugin)
+
+lazy val `hello` = (project in file("."))
+  .aggregate(`hello-api`, `hello-impl`)
+
+lazy val `hello-api` = (project in file("hello-api"))
+  .settings(
+    libraryDependencies ++= Seq(
+      lagomScaladslApi
+    )
+  )
+
+val check = taskKey[Unit]("Task for verifying Dockerfile labels")
+lazy val `hello-impl` = (project in file("hello-impl"))
+  .enablePlugins(LagomScala, SbtReactiveAppPlugin)
+  .settings(
+    packageName in Docker := "hello-lagom",
+    dockerExposedPorts in Docker := Seq(9000),
```

Comment:
I think you want `httpIngressPorts` here if you're intending to change the value of e.g. `LABEL com.lightbend.rp.endpoints.0.ingress.0.ingress-ports.1="443"`